var/spack/repos/builtin/packages/axom/package.py

```diff
@@ -93,6 +93,7 @@ class Axom(CachedCMakePackage, CudaPackage, ROCmPackage):
     # Basics
     depends_on("cmake@3.8.2:", type="build")
     depends_on("cmake@3.16.8:", type="build", when="+rocm")
+    depends_on("cmake@3.21:", type="build", when="@0.7.0")
```

Comment:
```suggestion
    depends_on("cmake@3.14:", type="build")
    depends_on("cmake@3.21:", type="build", when="+rocm")
```

Reply:
I would have to go back through releases to find a more accurate check but I am pretty sure this is okay for quite a few releases back.